core/src/trezor/messages.py

```
@@ -24,9 +24,10 @@ def __getattr__(name: str) -> Any:
     from trezor.enums import ButtonRequestType  # noqa: F401
     from trezor.enums import Capability  # noqa: F401
     from trezor.enums import CardanoAddressType  # noqa: F401
+    from trezor.enums import CardanoCVoteRegistrationFormat  # noqa: F401
```

Ruff (0.8.2)

Unused `noqa` directive (unused: `F401`)

Remove unused `noqa` directive

(RUF100)